### ISSUE TITLE
Installation error regarding soulutf8 on CTAN

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [pfmc-assessments/sa4ss#82]
**Describe how far you got**
- [x] installed tinytex
- [x] installed pandoc
- [x] installed sa4ss
- [ ] ran sa4ss::session_test()
- [ ] set up my own draft
- [ ] compiled my own document

**Place output from `sa4ss::session_info()` here if possible**
```
Warning: Package soulutf8 Warning: This package is obsolete,
Warning: (soulutf8)                use the soul package directly.
Warning: (soulutf8)                 on input line 29.
```

**Comment by kellijohnson-NOAA** [comment]
[soul](https://ctan.org/pkg/soul?lang=en) was recently updated on February 19, 2023 so I am hoping that this will magically go away eventually.

**Comment by kellijohnson-NOAA** [comment]
The errors are changed but still present as of April 2023
```
Warning: Package soulutf8 Warning: Command \SOUL@eval has changed.
Warning: (soulutf8)                Supported versions of package `soul': 2003/11/17.
Warning: (soulutf8)                Depending on the unknown changes the redefinition
Warning: (soulutf8)                of \SOUL@eval may not behave correctly.
```

**Comment by kellijohnson-NOAA** [comment]
FYI removing the calls to polyglossia removed this warning. See [stack exchange](https://tex.stackexchange.com/questions/517015/conflict-between-package-pdfcomment-and-polyglossia-during-compilation-by-lualat) for how I found this out.

**Comment by kellijohnson-NOAA** [comment]
Fixed with 98bd267
